### PR TITLE
Topic recognition-implement improved unet for OASIS brain dataset s4563609

### PR DESCRIPTION
**Pull request by dannyt622**
Hi Shakes,

Name: HoYin Tong
Student ID: 4563609

Here is my report for COMP3710. I am doing question 1 implement improved unet for OASIS brain dataset. This repo will contain a file call improved_unet and the file should contain 2 scripts (drive and model). And I have archived Dice similarity coefficient of >0.9.

Thanks!

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Most of the commits were made after deadline
- code appear okay for Easy Difficult
- results are good

*Other feedback*:
- include more details in commit messages
- comment your code, there are no comments in the model file
- credit any tutorial you may have used

**Comment by rexsw** [comment]
Code looks good for normal difficulty

For next time;

Code in files like driver scripts should be in a main function

File: recognition/improved_unet/model.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import tensorflow as tf
```
Comment (shakes76) [inline recognition/improved_unet/model.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. Resolve conflict for Recognition folder readme and main readme, it shouldn't change. Otherwise I can't merge into main repo**
No Proper Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Comment by dannyt622** [comment]
Hi Shakes,
Thanks for the feedback, I have solved those conflicts.

**Comment by shakes76** [comment]
You need to revert the changes to recognition/README.md as it overwrites current project readme

**Comment by dannyt622** [comment]
Added back recognition/README.md.
Just wondering will I still get the feedback marks(addressing feedback)?

**Pull request merged**